app/scripts/lib/account-tracker.js

```diff
@@ -281,6 +282,13 @@ export default class AccountTracker {
           );
           break;
 
+        case CHAIN_IDS.CONSENSYS_ZKEVM:
```

Comment:
we probably shouldn't include this as `SINGLE_CALL_BALANCES_ADDRESS_CONSENSYS_ZKEVM` is an empty string right now